worker/src/RTC/WebRtcServer.cpp

```diff
@@ -97,11 +97,11 @@ namespace RTC
 			auto jsonPortIt = jsonListenInfo.find("port");
 
 			if (jsonPortIt == jsonListenInfo.end())
```

Comment:
These conditions are a bit hard to read. We usually handle first the case in which value is properly given, then lack of given value (so ser port to 0), then the throw due to invalid value.